### ISSUE TITLE
External CSS with absolute url not generate correctly.

### ISSUE THREAD

**Issue by stanleybz** [addyosmani/critical#395]
Hi, I'm trying to generate CCSS on my site which containing an external CSS from my CDN, I found the background image or font URL is not correct if the CSS is using an absolute path, here's an example:

Main site: http://rcvc.io/test-ccss/
CDN CSS: http://cdn.rcvc.io/css/style.css

CCSS result:
```
.relative {
  background-image: url("http://cdn.rcvc.io/image.jpeg");
  font-size: 100px;
}
.absolute {
  background-image: url("http://cdn.rcvc.io/css/image.jpeg");
  font-size: 100px;
}
```

You can see if it's using relative path (**http://cdn.rcvc.io/image.jpeg**) the URL will be generated correctly but absolute path (**http://cdn.rcvc.io/css/image.jpeg**) not

The absolute path added an extra subfolder path(**/css**) which is the CSS file located.

If I use `rebase: asset => asset.url` the CDN origin will be lost, any solution for this?

**Comment by bezoerb** [comment]
@stanleybz: Can you post your complete critical config as well as the <head> part of your html?

**Comment by stanleybz** [comment]
Hi @bezoerb, 

Here's my config:
```
{
	src: inputURL,
	ignore: {
		atrule: ['@font-face', '@import'],
	},		
	minify: false,
	userAgent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1',
	width: is_mobile ? 479 : 1366,
	height: is_mobile ? 812 : 768,
}
```

Here's the full critical generator function: https://gist.github.com/stanleybz/072af59682b2a877435592a16dd8c285

Version: `"critical": "^2.0.0-18"`

**Comment by bezoerb** [comment]
@stanleybz: Sorry for the late response. I published a new version which should fix this issue [v2.0.0-22](https://github.com/addyosmani/critical/releases/tag/v2.0.0-22). Do you want to give another try?

**Comment by stanleybz** [comment]
Thank you @bezoerb! Just tested and it is working now.
